mvapi/settings.py

```diff
@@ -82,7 +82,8 @@
         'ENGINE': 'django.db.backends.postgresql',
         'NAME': 'postgres',
         'USER': 'postgres',
-        'HOST': 'db',
+        'HOST': 'localhost',
+        'PASSWORD': 'postgres',
```

Comment:
Did you need this? On my machine it works without it...

Reply:
Right it is because I can't have Docker working fine here. I will remove those lines once my PR is ready